.github/workflows/main.yml

```
@@ -0,0 +1,42 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ main, setup-github-actions ]
```

This shouldn't be necessary now that a PR has been created for the `main` branch:
```suggestion
    branches: [ main ]
```